### PR TITLE
Feature/events page lite

### PR DESCRIPTION
**Pull request by mdvanes**
Martin, this is the very minimal version of the Events page. We need it for the next Meetup. There is also a more elaborate branch feature/events-page that contains the full version and many (todo's) for fixes.

Can you check if this is good enough for now?

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
+// Mocked because of UnhandledPromiseRejectionWarning in Events
+jest.mock('./Events/Events', () => () => <div />);
```
Comment (mvdam) [inline src/App.test.js]: If we add a `catch` to that promise, could this then be removed?

File: src/Events/Events.jsx [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+// Meetup API test console: https://secure.meetup.com/meetup_api/console/?path=/:urlname/events
+// page=3 = number of results to return in a page, only need the first 3 results
+const GET_UPCOMING_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?photo-host=public&page=3&sig_id=226887185&status=upcoming&fields=featured_photo&sig=d035ab8a8f521cbb4ef14eaff79a55f23c3d25eb';
+
+const GET_PAST_EVENTS_URL =
+	'https://api.meetup.com/Code-Star-Night/events?desc=true&photo-host=public&sig_id=226887185&status=past&fields=featured_photo&sig=a60e663f0904424f80fda3b00bf31f315889231c';
+
+const styles = {
```
Comment (mvdam) [inline src/Events/Events.jsx]: We can also put this in a `.css` file to keep the component tidy?

File: src/Events/Events.jsx [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+		this.fetchEvents = this.fetchEvents.bind(this);
+		this.fetchEvents();
+	}
+
+	fetchEvents() {
+		/* Meetup API only allows JSONP for client-side, non authenticated, api key signed GET requests.
+		   must use JSONP conform https://github.com/meetup/api/issues/211
+		   Fetch API does not support JSONP. no-cors mode creates an opaque response without data.
+		*/
+		jsonp(GET_UPCOMING_EVENTS_URL).then(response => {
```
Comment (mvdam) [inline src/Events/Events.jsx]: Ideally we should fetch things in container components. What do you think?

File: src/App.test.js (hunk 1 above)
Comment (mdvanes) [reply src/App.test.js]: Agreed, will fix in feature/events-page

File: src/Events/Events.jsx (hunk 2 above)
Comment (mdvanes) [reply src/Events/Events.jsx]: Agreed, will fix in feature/events-page

File: src/Events/Events.jsx (hunk 3 above)
Comment (mdvanes) [reply src/Events/Events.jsx]: Agreed, will fix in feature/events-page

**Pull request merged**